### PR TITLE
Update monorail.yml definitions

### PR DESCRIPTION
**Pull request by jlongever**
Generate 1.1 API client library from YAML definitions, remove apidoc-swagger dependency

@RackHD/corecommitters @WangWinson 
This removes the apidoc-swagger dependency and generates 1.1 client code from static/monorail.yml.  This keeps us consistent,  and probably should have been done this way originally;  it will also account for recent workflow api changes.   Once this goes in I can update pypi and submit the relative RackHD integration script changes, which are ready to go.

### CODE REVIEW

**Comment by jlongever** [comment]
fyi:  pending integration script changes after this goes in:  https://github.com/jlongever/RackHD/commit/8ed616cfba8c46c6f1099ef588cbcee800ff76bc

**Comment by WangWinson** [comment]
:+1: Finally, we have an official place to update security definitions

File: .travis.yml [hunk 1]
```diff
@@ -27,12 +27,11 @@ after_success:
 
 before_deploy:
  - npm run apidoc
- - npm run apidoc-swagger
  - npm prune --production
  - sudo apt-get install -y maven
  - git clone --branch v2.1.5 https://github.com/swagger-api/swagger-codegen.git
  - pushd ./swagger-codegen && mvn package && popd
- - java -jar ./swagger-codegen/modules/swagger-codegen-cli/target/swagger-codegen-cli.jar generate -i build/apidoc-swagger/swagger.json -o on-http-api1.1 -l python --additional-properties packageName=on_http
+ - java -jar ./swagger-codegen/modules/swagger-codegen-cli/target/swagger-codegen-cli.jar generate -i static/monorail.yml -o on-http-api1.1 -l python --additional-properties packageName=on_http_api1_0
```
Comment (anhou) [inline .travis.yml]: ‘on_http_api1_0’ to 'on_http_api1_1’?

**Comment by anhou** [comment]
:+1:

**Comment by anhou** [comment]
@jlongever just confirm that static/monorail.yml is generated manually, right?

File: .travis.yml (hunk 1 above)
Comment (jlongever) [inline .travis.yml]: Lol. Nice catch!

**Comment by jlongever** [comment]
@anhou AFAIK,  the 1.1 monorail.yml was generated manually.

**Comment by jlongever** [comment]
@anhou no, apidoc-swagger did not generate the file, it was generated well before I introduced apidoc-swagger.

**Comment by jlongever** [comment]
test this please

**Comment by WangWinson** [comment]
@jlongever 
I tried to generate python client code use `monorail.yml` locally. But it failed. Not sure if you have a try ?

**Pull request merged**